api-catalog-ui/frontend/src/components/Header/HeaderDetail.jsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+import { Component } from 'react';
+
+// Component provided as placeholder
+// NOSONAR
+export default class HeaderDetail extends Component {
```

Comment:
Is this placeholder needed even with the logic in `ServiceNavigationBar`?

Reply:
I fount it is.